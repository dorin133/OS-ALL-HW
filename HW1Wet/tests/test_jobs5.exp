smash> smash> smash> smash> [1] sleep 100& : 111003 0 secs
[2] sleep 200& : 111004 0 secs
[3] sleep 300& : 111005 0 secs
smash> smash> signal number 9 was sent to pid 111004
smash> smash> [1] sleep 100& : 111003 3 secs
[3] sleep 300& : 111005 3 secs
smash> signal number 9 was sent to pid 111003
smash> signal number 9 was sent to pid 111005
smash> smash> smash> smash: sending SIGKILL signal to 0 jobs:
