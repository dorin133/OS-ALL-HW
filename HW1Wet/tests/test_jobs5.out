smash> smash> smash> smash> [1] sleep 100& : 111003 0 secs
[2] sleep 200& : 111006 0 secs
[3] sleep 300& : 111009 0 secs
smash> smash> signal number 9 was sent to pid 111006
smash> smash> [1] sleep 100& : 111003 3 secs
[3] sleep 300& : 111009 3 secs
smash> signal number 9 was sent to pid 111003
smash> signal number 9 was sent to pid 111009
smash> smash> smash> smash: sending SIGKILL signal to 0 jobs:
